config/configtls/configtls.go

```diff
@@ -17,9 +17,8 @@ package configtls // import "go.opentelemetry.io/collector/config/configtls"
 import (
 	"crypto/tls"
 	"crypto/x509"
-	"errors"
 	"fmt"
-	"os"
+	"io/ioutil"
```

Comment:
FYI, ioutil is deprecated and its functionality moved into the os and io packages